geometry_msgs/PoseStamped object_pose
string object_name
string[] allow_contact_with
---
#result definition
# Comes from moveit_msgs/MoveItErrorCodes
int32 error_code
---
#feedback
